elements/chameleon-common/post-install.d/50-openrc

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+if [ ${DIB_DEBUG_TRACE:-0} -gt 0 ]; then
+    set -x
+fi
+set -eu
+set -o pipefail
+
+# script for generating openrc file
+cat > /etc/auto_generate_openrc <<- EOM
```

Comment:
There's a little trick you can do to avoid having to do the `\$` everywhere.

```shell
cat > /etc/auto_generate_openrc <<- 'EOM'
.. stuff
EOM
```

Putting quotes around EOM is the trick.

See https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_07_04

It's a very unintuitive interface ;)

Reply:
good to know!